### PR TITLE
Use <For/> in `ColorPicker` documentation

### LINKED ISSUE #3731: Use `<For />` in `ColorPicker` documentation
### Provide the exact quote of the error or issue
We should use `<For />` component when repeating a component
e.g.
```tsx
<For each={items}>{(item, index) => <Text key={index} fontWeight="bold">{item}</Text>}</For>
```

https://yamada-ui.com/components/forms/color-picker

### Does the docs page already exist? Please link to it.

_No response_

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### LINKED ISSUE #3744: Use `<For />` in `List` documentation
### Provide the exact quote of the error or issue
We should use `<For />` component when repeating a component
e.g.
```tsx
<For each={items}>{(item, index) => <Text key={index} fontWeight="bold">{item}</Text>}</For>
```

https://yamada-ui.com/components/data display/list

### Does the docs page already exist? Please link to it.

_No response_

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### PR DESCRIPTION
**Pull request by kaehehehe**
Closes #3731

## Description

Fixed the repetition part in the `ColorPicker` documentation to use the `For` component.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by 108yen — DISMISSED** [review]

File: docs/contents/components/forms/color-picker/index.ja.mdx [hunk 1]
```diff
@@ -210,9 +274,22 @@ import { ColorPicker } from "@yamada-ui/react"
 
 ```tsx
 <VStack>
-  <ColorPicker isDisabled variant="outline" placeholder="outline" />
-  <ColorPicker isDisabled variant="filled" placeholder="filled" />
-  <ColorPicker isDisabled variant="flushed" placeholder="flushed" />
+  <For
+    each={[
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/color-picker/index.ja.mdx:278]: Change `each={["outline", "filled", "flushed"]}`.

File: docs/contents/components/forms/color-picker/index.ja.mdx [hunk 2]
```diff
@@ -222,9 +299,22 @@ import { ColorPicker } from "@yamada-ui/react"
 
 ```tsx
 <VStack>
-  <ColorPicker isReadOnly variant="outline" placeholder="outline" />
-  <ColorPicker isReadOnly variant="filled" placeholder="filled" />
-  <ColorPicker isReadOnly variant="flushed" placeholder="flushed" />
+  <For
+    each={[
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/color-picker/index.ja.mdx:303]: Change `each={["outline", "filled", "flushed"]}`.

File: docs/contents/components/forms/color-picker/index.mdx
Comment (hirotomoyamada) [inline docs/contents/components/forms/color-picker/index.mdx:1]: Please modify it as you did for the Japanese page.

File: docs/contents/components/forms/color-picker/index.ja.mdx [hunk 3]
```diff
@@ -28,32 +28,96 @@ import { ColorPicker } from "@yamada-ui/react"
 
 ```tsx
 <VStack>
-  <ColorPicker variant="outline" placeholder="outline" />
-  <ColorPicker variant="filled" placeholder="filled" />
-  <ColorPicker variant="flushed" placeholder="flushed" />
-  <ColorPicker variant="unstyled" placeholder="unstyled" />
+  <For
+    each={[
```
[the remark this answers is not in the document]
Comment (kaehehehe) [reply docs/contents/components/forms/color-picker/index.ja.mdx:32]: Fixed.

File: docs/contents/components/forms/color-picker/index.ja.mdx (hunk 2 above)
Comment (kaehehehe) [reply docs/contents/components/forms/color-picker/index.ja.mdx:303]: Fixed.

File: docs/contents/components/forms/color-picker/index.ja.mdx (hunk 1 above)
Comment (kaehehehe) [reply docs/contents/components/forms/color-picker/index.ja.mdx:278]: Fixed.

File: docs/contents/components/forms/color-picker/index.ja.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (odendayoko) [reply docs/contents/components/forms/color-picker/index.ja.mdx]: @kaehehehe 
Add `unstyled` to the array.

**Review by odendayoko — CHANGES_REQUESTED** [review]
@kaehehehe 
Please make similar modifications.
```diff
- <For each={["outline", "filled", "flushed"]}>
+ <For each={["outline", "filled", "flushed", "unstyled"]}>
```

- `docs/contents/components/forms/color-picker/index.ja.mdx` L31
- `docs/contents/components/forms/color-picker/index.mdx` L33

**Comment by kaehehehe** [comment]
> @kaehehehe Please make similar modifications.
> 
> ```diff
> - <For each={["outline", "filled", "flushed"]}>
> + <For each={["outline", "filled", "flushed", "unstyled"]}>
> ```
> 
> * `docs/contents/components/forms/color-picker/index.ja.mdx` L31
> * `docs/contents/components/forms/color-picker/index.mdx` L33

@odendayoko 
Thank you for catching this mistake. :)

**Review by odendayoko — APPROVED** [review]
@kaehehehe 
Thanks!! LGTM🧡

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**